### ISSUE TITLE
YubiKey Manager not displaying GUI

### ISSUE THREAD

**Issue by John-Hartman** [Yubico/yubikey-manager-qt#260]
- Yubico Authenticator version: 
- Operating system and version: 
- YubiKey model and version: 
- Bug description summary: 

Steps to reproduce
I had used it once to originally set up my YubiKey but ever since it has been set up I haven't needed to mess with it. Now I'm required to update my certs but the manager will not display. I have tried rebooting, uninstalling, and the commands from Yubico/yubioath-desktop#673 and Yubico/yubioath-desktop#678 . The problem is consistent and recreates every time I try to open the manager.

[Please explain what you did when the bug appeared, and if and how you have been
able to reproduce it.]

Expected result
I expected the Manager GUI to display so that I could use it.

[What did you expect to happen when you did the above?]

Actual results
When the app is opened it displays the Yubico Icon on the taskbar but there is no window that displays on the screen. I can hover over the Icon and it will show the preview of the window but it's just a blank white square.

[What actually happened?]

Other info
As noted above this is very similar to issues Yubico/yubioath-desktop#673 and Yubico/yubioath-desktop#678 however I noticed that one of the solutions from 678 was to run a set of commands. Unfortunately when I navigate to C://Program Files/Yubico I only have Yubikey Manager as a subfolder. I do not have Yubikey Authenticator. When I ran the below commands:
`C:\Program Files\Yubico\YubiKey Manager>set QT_OPENGL=software`

`C:\Program Files\Yubico\YubiKey Manager>ykman-gui.exe`

I got an error the first time but now I cannot recreate it. By running the commands it simply pulls up the same issue, a taskbar icon with a blank white square.

[Anything else you would like to add?]

**Comment by John-Hartman** [comment]
2021-02-23T10:07:20-0500 INFO [ykman.logging_setup.setup:58] Initialized logging for ykman version: 3.1.1
2021-02-23T10:07:20-0500 DEBUG [ykman.descriptor.Descriptor.open_device:87] transports: 0x7, self.mode.transports: 0x4
2021-02-23T10:07:20-0500 DEBUG [ykman.descriptor.open_device:92] Opening driver for serial: None, type: YUBIKEY.YK4, mode: CCID
2021-02-23T10:07:20-0500 DEBUG [ykman.descriptor.open_device:95] Attempt 1 of 10
2021-02-23T10:07:20-0500 DEBUG [ykman.descriptor.open_device:98] Found driver: <ykman.driver_ccid.CCIDDriver object at 0x000001AD9166CFD0>, key_type: YUBIKEY.YK4, mode: CCID
2021-02-23T10:07:20-0500 DEBUG [ykman.device.__init__:198] Read config from device...
2021-02-23T10:07:20-0500 DEBUG [ykman.driver_ccid.send_apdu:228] SEND: b'00a4040008a000000527471117'
2021-02-23T10:07:20-0500 DEBUG [ykman.driver_ccid.send_apdu:230] RECV: b'5669727475616c206d6772202d2046572076657273696f6e20342e342e359000'
2021-02-23T10:07:20-0500 DEBUG [ykman.driver_ccid.send_apdu:228] SEND: b'001d000000'
2021-02-23T10:07:20-0500 DEBUG [ykman.driver_ccid.send_apdu:230] RECV: b'1401013f020400be0efd03013c04010005030404059000'
2021-02-23T10:07:20-0500 DEBUG [ykman.device.__init__:200] Success!
2021-02-23T10:07:20-0500 DEBUG [ykman.driver_ccid.close:270] Close <ykman.driver_ccid.CCIDDriver object at 0x000001AD9166CFD0>
2021-02-23T10:07:20-0500 DEBUG [ykman.driver_ccid.__del__:275] Destroy <ykman.driver_ccid.CCIDDriver object at 0x000001AD9166CFD0>

**Comment by fdennis** [comment]
Thanks! Looks like the backend is working correctly. However, for some reason the GUI does not show for you. This does indeed look similar to the issues you have linked to and, unfortunately, there can be many different causes for this.. If you have followed the advice given there and it has not helped, then perhaps this might be an issue with your drivers. Another solution might be to downgrade to another, older version of YubiKey Manager GUI and see if that works.

We do also have a CLI that comes bundles with the GUI. You can find it in the same folder and is simply called `ykman.exe`. If nothing else works, I would suggest that you use the CLI as a workaround. There is a guide available [here](https://support.yubico.com/hc/en-us/articles/360016614940-YubiKey-Manager-CLI-ykman-User-Manual) on how to use it.

**Comment by dainnilsson** [comment]
Since you mentioned there is no window shown at all, I would suspect it's being positioned "off screen". Take a look at this issue https://github.com/Yubico/yubioath-desktop/issues/241 and see if deleting the registry entries for position (while the app is closed, then launch it) help.

**Comment by John-Hartman** [comment]
That's a pretty good point, when I hover over it every other window dissapears and I just see my desktop. I'm running three monitors and I'm opening it from my second monitor.

**Comment by John-Hartman** [comment]
So I was able to resolve the issue:

@dainnilsson was right when he suggested that it was a display issue. 

Solution:
Unplug all external monitors and then reboot the computer (laptop in my case) leaving all of the monitors unplugged. Opened the YubiKey manager from my laptop and did what I had to do. Closed the application before plugging my monitors back in.

I plugged the monitors back in and tried opening the app again from my laptop screen and it opened just fine.
